sdc/datatypes/hpat_pandas_series_rolling_functions.py

```diff
@@ -625,6 +625,9 @@ def impl(self):
                 nfinite, result = pop(pop_value, nfinite, result)
                 output_arr[idx] = get_result(nfinite, minp, result)
 
+        if minp > 1:
+            output_arr[:minp-1] = numpy.nan
```

Comment:
Doesn't it break other tests on rolling?

Reply:
@densmirn  I run all tests, all OK.